### PR TITLE
Add github address

### PR DESCRIPTION
**Pull request by ragingwind**
It would be helpful

### CODE REVIEW

File: codebusking.json [hunk 1]
```diff
@@ -2,5 +2,6 @@
 	"firstName": "Code",
 	"lastName": "Busking",
 	"email": "me@codebusking.com",
-	"homepage": "www.codebusking.com"
+	"homepage": "www.codebusking.com",
+	"github": "http://github.com/codebusking"
```
Comment (azurepeak) [inline codebusking.json]: HTTPS, not HTTP

File: test.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ test('test properties', t => {
 	t.is(codeBusking.lastName, 'Busking');
 	t.is(codeBusking.email, 'me@codebusking.com');
 	t.is(codeBusking.homepage, 'www.codebusking.com');
+	t.is(codeBusking.github, 'http://github.com/codebusking');
```
Comment (azurepeak) [inline test.js]: Same as above

File: codebusking.json (hunk 1 above)
Comment (ragingwind) [inline codebusking.json]: Oh, my bad. I will update

File: codebusking.json [hunk 3]
```diff
@@ -3,5 +3,5 @@
 	"lastName": "Busking",
 	"email": "me@codebusking.com",
 	"homepage": "www.codebusking.com",
-	"github": "http://github.com/codebusking"
+	"github": "https://github.com/codebusking"
```
Comment (azurepeak) [inline codebusking.json]: 👍

**Comment by azurepeak** [comment]
Squash your commits

**Pull request merged**